src/migrations.zig

```diff
@@ -26,28 +26,44 @@ fn register(
 ) void {
 	const localAllocator = main.stackAllocator;
 
-	if((migrationZon != .object or migrationZon.object.count() == 0)) {
+	if((migrationZon != .array or migrationZon.array.items.len == 0)) {
```

Comment:
Not a requirement, you can decide if you want to use this:
This could also be reduced to just `migrationZon.toSlice().len == 0`, since for all other types that function returns a slice of length 0.

Reply:
Until then I've just removed the empty file.